dallinger/experiment_server/experiment_server.py

```diff
@@ -319,7 +319,10 @@ def advertisement():
         status = part.status
     except exc.SQLAlchemyError:
         status = None
-
+    try:
+        app_id = config.get('id')
```

Comment:
Like with a Python dictionary, you can specify a default value as the second argument to the configuration's get method:

```
app_id = config.get('id', 'unknown')
```